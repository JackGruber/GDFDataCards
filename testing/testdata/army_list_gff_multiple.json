{
    "id": "MChs07FpkJlF",
    "gameSystem": "gff",
    "armyId": "jpc0kyil0juwy602",
    "armyIds": [
        "jpc0kyil0juwy602",
        "7oi8zeiqfamiur21"
    ],
    "armyFaction": null,
    "armyName": "Human Inquisition",
    "modified": "2024-10-27T16:36:40.298Z",
    "saveVersion": 3,
    "favourite": false,
    "armyVersions": [
        {
            "version": "3.4.1",
            "armyId": "jpc0kyil0juwy602"
        },
        {
            "version": "3.4.1",
            "armyId": "7oi8zeiqfamiur21"
        }
    ],
    "listPoints": 110,
    "list": {
        "id": "MChs07FpkJlF",
        "key": "Nw2radexergChnUDXoP4B",
        "name": "Human Inquisition",
        "description": "",
        "pointsLimit": 300,
        "campaignMode": false,
        "narrativeMode": false,
        "forceOrg": true,
        "gameSystem": "gff",
        "isCloud": false,
        "modified": "2024-10-27T16:36:40.298Z",
        "cloudModified": null,
        "activationCount": 3,
        "modelCount": 3,
        "units": [
            {
                "id": "ZaWWIux",
                "armyId": "jpc0kyil0juwy602",
                "selectionId": "ebHD5",
                "selectedUpgrades": [],
                "combined": false,
                "joinToUnit": null,
                "xp": 0,
                "traits": [],
                "notes": null
            },
            {
                "id": "e9dUXdy",
                "armyId": "jpc0kyil0juwy602",
                "selectionId": "mVLWh",
                "selectedUpgrades": [],
                "combined": false,
                "joinToUnit": null,
                "xp": 0,
                "traits": [],
                "notes": null
            },
            {
                "id": "2TXWi3B",
                "armyId": "7oi8zeiqfamiur21",
                "selectionId": "bKaxX",
                "selectedUpgrades": [],
                "combined": false,
                "joinToUnit": null,
                "xp": 0,
                "traits": [],
                "notes": null
            }
        ]
    }
}