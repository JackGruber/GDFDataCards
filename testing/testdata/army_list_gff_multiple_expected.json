{
    "armyId": "jpc0kyil0juwy602",
    "gameSystem": "gff",
    "gameSystemId": 3,
    "armyVersions": [
        {
            "version": "3.4.1",
            "armyId": "jpc0kyil0juwy602"
        },
        {
            "version": "3.4.1",
            "armyId": "7oi8zeiqfamiur21"
        }
    ],
    "armyName": "Human Inquisition",
    "listPoints": 110,
    "listName": "Human Inquisition",
    "units": [
        {
            "type": "Inquisitor",
            "name": "Inquisitor",
            "armyId": "jpc0kyil0juwy602",
            "id": "ZaWWIux",
            "cost": 55,
            "defense": 3,
            "quality": 3,
            "upgrades": [
                "A1"
            ],
            "size": 1,
            "notes": null,
            "weapons": [
                {
                    "attacks": 1,
                    "count": 1,
                    "name": "Heavy Pistol",
                    "range": 12,
                    "specialRules": [],
                    "ap": 1
                },
                {
                    "attacks": 2,
                    "count": 1,
                    "name": "CCW",
                    "specialRules": []
                }
            ],
            "equipment": [],
            "rules": [
                {
                    "id": "2atvlmICQSCV",
                    "name": "Fearless",
                    "label": "Fearless"
                },
                {
                    "id": "T08VkBuVmNZ_",
                    "name": "Hero",
                    "label": "Hero"
                },
                {
                    "id": "a0YtInGiUDd6",
                    "name": "Tough",
                    "rating": 3,
                    "label": "Tough(3)"
                }
            ]
        },
        {
            "type": "Cyborg Henchman",
            "name": "Cyborg Henchman",
            "armyId": "jpc0kyil0juwy602",
            "id": "e9dUXdy",
            "cost": 20,
            "defense": 4,
            "quality": 5,
            "upgrades": [
                "JFTXK"
            ],
            "size": 1,
            "notes": null,
            "weapons": [
                {
                    "attacks": 2,
                    "count": 1,
                    "name": "Cyborg Arm",
                    "specialRules": [],
                    "ap": 4
                }
            ],
            "equipment": [],
            "rules": []
        },
        {
            "type": "High Sister",
            "name": "High Sister",
            "armyId": "7oi8zeiqfamiur21",
            "id": "2TXWi3B",
            "cost": 35,
            "defense": 4,
            "quality": 4,
            "upgrades": [
                "A1",
                "B1",
                "C1"
            ],
            "size": 1,
            "notes": null,
            "weapons": [
                {
                    "attacks": 2,
                    "count": 1,
                    "name": "CCW",
                    "specialRules": []
                }
            ],
            "equipment": [
                {
                    "name": "Combat Shield",
                    "specialRules": [
                        {
                            "id": "wsqfB0fq69eG",
                            "name": "Shield Wall",
                            "type": "ArmyBookRule",
                            "label": "Shield Wall"
                        }
                    ]
                }
            ],
            "rules": [
                {
                    "id": "j_GPMzrugrCj",
                    "name": "Devout",
                    "additional": false,
                    "label": "Devout"
                },
                {
                    "id": "T08VkBuVmNZ_",
                    "name": "Hero",
                    "label": "Hero"
                },
                {
                    "id": "a0YtInGiUDd6",
                    "name": "Tough",
                    "rating": 3,
                    "label": "Tough(3)"
                }
            ]
        }
    ]
}